api/src/main/java/ca/bc/gov/educ/api/graduation/util/EducGraduationApiUtils.java

```diff
@@ -107,6 +107,19 @@ public static String formatDateForReportJasper(String updatedTimestamp) {
 
 	}
 
+	public static Date formatIssueDateForReportJasper(String updatedTimestamp) {
+		SimpleDateFormat fromUser = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
Consider using a constant for "yyyy-MM-dd" or a method for getting a SimpleDateFormat with a consistent type (like getSimpleDateFormat() which returns one formatted with "yyyy-MM-dd".

Reply:
Moved to Utils